docs/decisions/0005-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,19 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django app. This method of granting permissions is not endorsed.
```

Comment:
```suggestion
Open edX administrators ocassionally need to grant permissions to staff users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django service. This method of granting permissions is not endorsed, since it is difficult review, audit, and track changes to user access over time.
```

Some suggested additional detail.